interface/pages/posts/new_person.tsx

```diff
@@ -30,92 +36,76 @@ export default function NewPerson() {
     validate: {
       name: (value) => (value ? null : 'Invalid name'),
       email: (value) => (/^\S+@\S+$/.test(value) ? null : 'Invalid email'),
-
     },
-
   });
 
   const postPerson = async (data: Omit<API.Person, "id">) => {
-    const response = await fetch(
-      `${URL}/persons/`,
-      {
-        method: 'POST',
-        headers: {
-          'Content-Type': 'application/json',
-        },
-        body: JSON.stringify(data),
-      }
-    )
-    const person = await response.json()
+    const response = await state.postPerson(data)
     if (response.status == 200) {
       console.log("POST /persons")
-      console.dir(person)
+      form.reset()
```

Comment:
This call to `form.reset` is performed after the data is successfully pushed to the db, hence it's a good place to reset the fields